### PR TITLE
Create DFRC-DREAMFLYF411.config

### PR DESCRIPTION
**Pull request by GEME-Cyi**
Add DFRC-DREAMFLYF411_support

### CODE REVIEW

**Comment by nerdCopter** [comment]
image

recommended git global environment:

windows:
```
git config --global core.autocrlf true
```
linux/osx:
```
git config --global core.autocrlf input
```
cross-platform:
```
git config --global core.safecrlf warn
git config --global core.whitespace cr-at-eol
git config --global core.filemode false
git config --global help.autocorrect true
```

File: configs/default/DFRC-DREAMFLYF411.config [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+
+
+# feature
+feature SOFTSERIAL
+feature LED_STRI
```
Comment (haslinghuis) [inline configs/default/DFRC-DREAMFLYF411.config:80]:
```suggestion
feature LED_STRIP
```

File: configs/default/DFRC-DREAMFLYF411.config [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+
+
+# feature
+feature SOFTSERIAL
```
Comment (haslinghuis) [inline configs/default/DFRC-DREAMFLYF411.config:79]:
```suggestion

```

No softserial pins defined

File: configs/default/DFRC-DREAMFLYF411.config [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+set current_meter = ADC
+set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set flash_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
```
Comment (haslinghuis) [inline configs/default/DFRC-DREAMFLYF411.config:101]: No newline at EOF
```suggestion
set gyro_1_sensor_align = CW270

```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**